app-frontend/src/store/sagas/worker.ts

```diff
@@ -38,3 +46,32 @@ export function* loadWorkerData(action: WorkerIdAction) {
     }
   }
 }
+
+export function* addWorkerData(action: WorkerAddAction) {
+  try {
+    axios({
+      method: 'POST',
+      url: `${API_URL}/workers`,
+      data: action.payload,
+      withCredentials: true,
+    });
+  } catch (error: unknown) {
+    if (error instanceof Error) {
+      yield put(addWorkerDataFailed(error.message));
+    }
+  }
+}
+
+export function* editWorkerData(action: WorkerEditAction) {
+  try {
+    axios({
+      method: 'PATCH',
+      url: `${API_URL}/workers/${action.payload.id}`,
+      data: action.payload.workerData,
+    });
+  } catch (error: unknown) {
+    if (error instanceof Error) {
+      yield put(editWorkerDataFailed(error.message));
+    }
+  }
+}
```

Comment:
It is better to attach sagas directly in the file of sagas.
Example:
`

export default function* watcherDealerFormsSaga(): WatcherSaga {
yield all([
takeEvery(REGISTRATION_DEALER.START, registrationDealerSaga),
takeEvery(FETCH_DEALER_FORM.START, fetchDealerFormSaga),
]);
}
`

Then in the sagas watcher you can simply do the following:
`
export default function* rootSaga() {
yield all([
watcherLanguageSaga(http),
watcherIntlSaga(),
...
]);
}

`